test/parallel/test-sqlite-statement-sync.js

```diff
@@ -168,7 +168,7 @@ suite('StatementSync.prototype.iterate()', () => {
     ]);
     t.assert.deepStrictEqual(
       iterator.next(),
-      { __proto__: null, done: true, value: null },
```

Comment:
Reviewer note: these are created as null-prototype dictionary objects, in which checking the `__proto__` is unnecessary